src/docfx/lib/log/Error.cs

```diff
@@ -24,14 +24,17 @@ internal class Error
 
         public bool PullRequestOnly { get; }
 
+        public object[] OverrideParameters { get; }
```

Comment:
>OverrideParameters [](start = 24, length = 18)

Can we simplify the error message overriding logic here. For example, we use this message to replace docfx default message, but we can still append additionalMessage when it is set?

Reply:
additionalMessage is used for metadata validation, here for learn content validation, we can config the error message directly, or maybe we will consider it more concretely in the future when we supporting error message UI configurable.